pkg/rosa/idps.go

```diff
@@ -0,0 +1,159 @@
+package rosa
+
+import (
+	"fmt"
+	"net/http"
+
+	cmv1 "github.com/openshift-online/ocm-sdk-go/clustersmgmt/v1"
+)
+
+func (c *RosaClient) ListIdentityProviders(clusterID string) ([]*cmv1.IdentityProvider, error) {
```

Comment:
Given all of these are exported, might be good to add godoc comments explaining what they do/are

Reply:
done